<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParticipantWindowClass</class>
 <widget class="QWidget" name="ParticipantWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ParticipantWindow</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>371</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QListWidget" name="listWidgetQuestions"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="labelAnswer">
        <property name="text">
         <string>Enter answer:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditAnswer"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonAnswer">
        <property name="text">
         <string>Answer</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="labelScore">
        <property name="text">
         <string>Score:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelValueScore">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ParticipantWindow.qrc"/>
 </resources>
 <connections/>
</ui>
